.github/workflows/stream-diff.yml

```diff
@@ -25,11 +25,28 @@ jobs:
           # Switch to base
           git checkout "${GITHUB_BASE_REF}"
 
+          # Get the numerically latest fedora-coreos-stream-generator tag
+          generator_tag=$(
+              curl -H 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' https://api.github.com/repos/coreos/fedora-coreos-stream-generator/git/matching-refs/tags/ |
```

Comment:
just curious.. is there no way to do this without authentication?

Reply:
It works fine without authentication, but has a lower rate limit.